### PR TITLE
created impress-common.css

### PR DESCRIPTION
**Pull request by IngridRegina**
Moved the css from src/plugins/readme.md to a css file(css/impress-common.css), removed duplicate code from the example css files and edited the example HTML files to use impress-common.css.
Should the readme.md also be edited accordingly?

### CODE REVIEW

File: examples/2D-navigation/css/presentation.css [hunk 1]
```diff
@@ -247,20 +237,6 @@ body.impress-on-overview #crisps-image {
...
-.impress-enabled div#impress-toolbar {
-    position: fixed;
-    right: 1px;
-    bottom: 1px;
-    opacity: 0.6;
-    z-index: 10;
-}
-.impress-enabled div#impress-toolbar > span {
-    margin-right: 10px;
-}
```
Comment (henrikingo) [inline examples/2D-navigation/css/presentation.css]: All of the below can be moved/removed too. Same for the other files.

File: css/impress-demo.css [hunk 2]
```diff
@@ -714,8 +700,7 @@ body.impress-mouse-timeout {
 */
 .impress-enabled                          { pointer-events: none }
 .impress-enabled #impress                 { pointer-events: auto }
```
[the remark this answers is not in the document]
Comment (IngridRegina) [reply css/impress-demo.css]: I moved the part to the common file.

File: css/impress-common.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
+/* Using the substep plugin, hide bullet points at first, then show them one by one. */
```
[the remark this answers is not in the document]
Comment (IngridRegina) [reply css/impress-common.css:7]: Added this as a comment at the top of the common file.

File: examples/2D-navigation/css/presentation.css (hunk 1 above)
Comment (IngridRegina) [reply examples/2D-navigation/css/presentation.css]: Should be done now.

**Comment by henrikingo** [comment]
Thanks for doing this!

**Pull request merged**